go/vt/logz/logz_utils.go

```diff
@@ -79,14 +79,33 @@ func StartHTMLTable(w http.ResponseWriter) {
                 }
 </style>
 
-<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.0/jquery.min.js"></script>
+<table class="gridtable">
+`))
+}
 
+// EndHTMLTable writes the end of a logz-style table to an HTTP response.
+func EndHTMLTable(w http.ResponseWriter) {
+	w.Write([]byte(`
+</table>
 <script type="text/javascript">
```

Comment:
The script was moved _after_ the `table` closing tag since on line 157, it expects table to be mounted onto the DOM already.